src/Disks/ObjectStorages/AzureBlobStorage/AzureObjectStorage.h

```diff
@@ -173,7 +173,8 @@ class AzureObjectStorage : public IObjectStorage
     MultiVersion<Azure::Storage::Blobs::BlobContainerClient> client;
     MultiVersion<AzureObjectStorageSettings> settings;
     const String object_namespace; /// container + prefix
-    const String description; /// url + container
+
+    const String description; /// source url without container and prefix
```

Comment:
```suggestion
    /// We use source url without container and prefix as description, because in Azure there are no limitations for operations between different containers.
    const String description;
```